queries/liquidityPools/useCurvePoolQuery_1.ts

```diff
@@ -144,6 +144,7 @@ const useCurvePoolQuery_1 = (options?: QueryConfig<CurveData>) => {
 				duration: Number(duration) * 1000,
 				allowance,
 				userBalance,
+				needsToSettle: false,
```

Comment:
you dont need to have this on all of them but its ok if you do. you can always do `needsToSettle?: boolean` when you define it instead.